include/openssl/x509v3.h

```diff
@@ -331,10 +331,6 @@ struct ISSUING_DIST_POINT_st {
 /* onlysomereasons present */
 # define IDP_REASONS     0x40
 
-# define X509V3_conf_err(val) ERR_add_error_data(6, \
```

Comment:
This is a breaking API change.

Reply:
Well, that macro is exported but undocumented.
Shall I move its definition back to `include/openssl/x509v3.h`?

Reply:
Just did this tentatively as well.